e2e/cypress/integration/enterprise/cloud/Billing/company_Information_spec.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only
+
+import {getRandomLetter} from '../../../../utils/index';
+
+describe('System Console - Company Information section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-37051 - Adding the Company Information screen', () => {
+        const compnayName = getRandomLetter(30);
+        gotoCompanyInformationScreen();
+
+        // # click on Add Company Information button
+        cy.contains('span', 'Add Company Information').parent().click();
+
+        // # enter company information functionality
+        cy.get('#input_companyName').clear().type(compnayName);
+        cy.get('#input_numEmployees').clear().type('10');
+        cy.get('.css-1hwfws3').click();
```

Comment:
this looks like a hashed class, if it is it will fail in the next build.

Reply:
can you please provide more details here , can I know what is the hash class you are referring here?

Reply:
I believe he's referring to `.css-1hwfws3` which we should not be using. Can we use `#DropdownInput_country_dropdown ` or if not maybe better to assign an ID to target element?

Reply:
Thanks, I have modified that with ID now.